### PR TITLE
Use GovukComponent::PhaseBanner

### PR DESCRIPTION
**Pull request by paulrobertlloyd**
### Context

* Uses `GovukComponent::PhaseBanner` to render phase banner component (via `govuk-components` `govuk_phase_banner` helper method).
* Renames `header_environment_class` to `environment_header_class`
* Renames `beta_banner_environment_label` to `environment_label`
* Replaces `beta_tag_environment_class` with `environment_colour`. This value is passed on to the tag in the phase banner component
* Renames `_beta-banner.html.erb` to `_phase-banner.html.erb`
* Moves `app-header` modifiers into `_header.scss`
* Removes `_environments.scss`

### Guidance to review

There should be no visual changes, aside from the environment tag, which now uses the standard (and legible) pastel tag colours from the design system rather than white text on solid colours. This is the same as used on Apply.

### Checklist

- [ ] Make sure all information from the Trello card is in here
- [ ] Attach to Trello card
- [ ] Rebased master
- [ ] Cleaned commit history
- [ ] Tested by running locally
- [ ] Product Review

### CODE REVIEW

File: app/helpers/view_helper.rb [hunk 1]
```diff
@@ -65,16 +65,29 @@ def provider_enrichment_error_url(provider:, field:)
...
+    case Settings.environment.selector_name
+    when "qa"
+      "orange"
+    when "staging"
+      "red"
+    when "development"
+      "grey"
+    when "review"
+      "purple"
+    when "unknown-environment"
```
Comment (defong) [inline app/helpers/view_helper.rb:80]: Can we have a pretty rollover one please 🙏

Comment (paulrobertlloyd) [reply app/helpers/view_helper.rb:91]: Done. How’s about this rather fetching turquoise?

Screenshot 2021-05-24 at 21 22 28

Comment (defong) [reply app/helpers/view_helper.rb:91]: Love it

**Review by defong — APPROVED** [review]

**Review by defong — APPROVED** [review]
@paulrobertlloyd is amazing

**Pull request merged**